src/message/MessageListItem.js

```diff
@@ -19,6 +19,8 @@ export default class MessageListItem extends PureComponent {
 
     if (type === 'time') {
       return <TimeRow key={`time${timestamp}`} timestamp={timestamp} />;
+    } else if (message.isOutbox) {
+      return <MessageContainer isBrief={isBrief} message={message} />;
```

Comment:
It get's picked up by the native component and comes in the list of visibleId's here https://github.com/zulip/zulip-mobile/blob/master/src/message/MessageListContainer.js#L22 and then there's an attempt to send this as an unread id!